client/src/Movies/MovieList.js

```diff
@@ -1,54 +1,57 @@
-import React, { Component } from 'react';
-import axios from 'axios';
+import React, { Component } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
 
 export default class MovieList extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      movies: []
-    };
-  }
+	constructor(props) {
+		super(props);
+		this.state = {
+			movies: []
+		};
+	}
 
-  componentDidMount() {
-    axios
-      .get('http://localhost:5000/api/movies')
-      .then(response => {
-        this.setState(() => ({ movies: response.data }));
-      })
-      .catch(error => {
-        console.error('Server Error', error);
-      });
-  }
+	componentDidMount() {
+		axios
+			.get("http://localhost:5000/api/movies")
+			.then(response => {
+				this.setState(() => ({ movies: response.data }));
+			})
+			.catch(error => {
+				console.error("Server Error", error);
+			});
+	}
 
-  render() {
-    return (
-      <div className="movie-list">
-        {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
-        ))}
-      </div>
-    );
-  }
+	render() {
+		return (
+			<div className="movie-list">
+				{this.state.movies.map(movie => (
+					<Link to={`/movies/${movie.id}`}>
```

Comment:
```suggestion
					<Link to={`/movies/${movie.id}`} key={movie.id}>
```
This component needs the key now that it is the outermost component being created from the map